frontend/src/lib/experimental/npsLogic.test.ts

```diff
@@ -0,0 +1,38 @@
+import { npsLogic } from 'lib/experimental/npsLogic'
+import { initKea } from '~/initKea'
+
+jest.mock('posthog-js')
+
+describe('NPS Logic', () => {
+    let unmount: () => void
+    beforeEach(() => {
+        initKea()
+        unmount = npsLogic.mount()
+    })
+    afterEach(() => {
+        unmount?.()
+    })
+
+    test('defaults', () => {
```

Comment:
This is better done as a snapshot test - you're just adding cruft here otherwise and making it annoying to update.`